samples/old/fleetprovisioning.py

```diff
@@ -0,0 +1,406 @@
+# Copyright 2010-2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
I don't see the value in having this old file around?
It just seems like a source of confusion
It lives forever in Git history

Reply:
I guess we can remove